src/workerd/io/compatibility-date.capnp

```diff
@@ -1066,13 +1066,14 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
      $compatEnableFlag("enable_nodejs_perf_hooks_module")
      $compatDisableFlag("disable_nodejs_perf_hooks_module")
      $experimental;
+     # $impliedByAfterDate(name = "nodeJsCompat", date = "2025-09-21");
    # Enables the Node.js perf_hooks module. It is required to use this flag with
    # nodejs_compat (or nodejs_compat_v2).
 
   enableGlobalPerformanceClasses @123 :Bool
      $compatEnableFlag("enable_global_performance_classes")
      $compatDisableFlag("disable_global_performance_classes")
-     $experimental;
+     $compatEnableDate("2025-10-01");
```

Comment:
Could this be reverted?
I think having "experimental" is the safest thing we can for now.
I have added a longer explanation on the PR.

Reply:
This is also fine, but this also needs an implied by attribute.
```suggestion
     $compatEnableDate("2025-10-01")
     $impliedByAfterDate(name = "enableNodeJsPerfHooksModule", date = "2025-10-01");
```